### DISCUSSION TITLE
Could I get some help hosting on render.com? Having some libreoffice problems

### DISCUSSION THREAD

**Discussion by ryanhuang519** [gotenberg/gotenberg#797] [Q&A]
Whenever I try to do libreoffice conversions, I always get:
`convert to PDF: process first start: start process: LibreOffice socket not available: context deadline exceeded`

I've also tried running with `autostart-libreoffice = true`, but my requests are now just timing out. Anyone able to help?

**Comment by gulien** [comment]
Hello @ryanhuang519,

The demo instance is working just fine on a « Starter » instance type. LibreOffice is also on auto-start. Otherwise only default options.

**Reply by ryanhuang519** [reply]
Ah, I think I might just be making a stupid mistake, I'm pretty new to Docker. Did you create it via their container hub link option? Or as a Docker repo that is connected. Also, how did you manage the GET "/" 200 health check? Also, how did you pass the auto-start-libreoffice option in Render? Thanks!

**Reply by gulien** [reply]
I have custom Dockerfile (based on the official image) with a slightly modified version of Gotenberg (e.g., a module that handles specific stuff for the demo). It is therefore connected via a GitHub repository (with a `render.yaml` file).

Other than that, I’m not really familiar with Render, but:

* `/health` is the correct health check URL
* You should look on how to override the Docker image command with Render
